site/pages/about-us/principles-slice/index.js

```diff
@@ -23,8 +22,8 @@ export default function Principles() {
           <Principle number="08" title="Give a monkey’s. Don’t be a jackass." description="We care about what we do and we care about each other. We’re humble. No egos." />
         </ul>
         <div className={style.buttons}>
-          <Link to="meetOurTeam" ><Button label="Meet our team" background="black" /></Link>
-          <Link to="joinUs"><Button label="Join us" background="black" /></Link>
+          <a href="/about-us/people/"><Button label="Meet our team" background="black" /></a>
+          <a href="/about-us/join-us/"><Button label="Join us" background="black" /></a>
```

Comment:
Join us can be a named route, as it exists in `website-honestly` so you can discard this line of the diff.